src/client/utils/stateContext.tsx

```diff
@@ -86,6 +87,13 @@ export const StateProvider = ({ children }: any) => {
     });
   };
 
+  //itentamos traer los portfolios
+  const getAllPortfolios = async () => {
+    const response = await callApi({ method: 'GET', endpoint: '/portfolios' });
+    console.log('response', response);
+    return response;
+  };
+
   // const getFile = async () => {

```

Comment:
I know you did not add this comments in the PR. but as general rule: 
`if is not in use, it should be removed`